mobile/src/components/setting/setting.tsx

```diff
@@ -35,6 +38,18 @@ const Settings: FC = () => {
     (state) => state.userDetails,
   );
 
+  const today = new Date();
+  const maximumDate = new Date(
+    today.getFullYear() - USER_MIN_AGE,
+    today.getMonth(),
+    today.getDay(),
+  );
+  const minimumDate = new Date(
+    today.getFullYear() - USER_MAX_AGE,
+    today.getMonth(),
+    today.getDay(),
+  );
```

Comment:
Create a helper in shared for this, use [differenceInYears](https://date-fns.org/v2.29.2/docs/differenceInYears) from date-fns. We decided on using only min age

Reply:
UPD: We will have it in this [PR](https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/375), wait till it will be merged

Reply:
PR was merged